src/vue-pdf-embed.vue

```diff
@@ -163,6 +163,9 @@ export default {
           const retry = reason === pdf.PasswordResponses.INCORRECT_PASSWORD
           this.$emit('password-requested', callback, retry)
         }
+        documentLoadingTask.promise.then((pdf) => {
+          this.$emit('loaded', pdf)
+        })
```

Comment:
This handler is not necessary, we can just emit `this.document` on the last line (after setting `this.pageCount`)

```js
this.$emit('loaded', this.document)
```

Reply:
Thank you for the tip. I tested this change and it works perfectly.